src/hooks/screen/useChatScreenHook.ts

```diff
@@ -58,6 +63,16 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       setTimeout(() => {
         setLoadingChat(false);
       }, 350);
+
+      if (type === 'ANONYMOUS') {
+        const userInfo = await UserSchema.getSelfAnonUserInfo(
+          localDb,
+          anonProfileId,
+          selectedChannel?.id
+        );
+
+        setSelfAnonUserInfo(userInfo);
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code block fetches anonymous user info when the chat type is 'ANONYMOUS'. This seems like a good approach for handling different chat types. However, ensure that error handling is in place for the async operation `UserSchema.getSelfAnonUserInfo`.
```diff
+ if (type === 'ANONYMOUS') {
+   const userInfo = await UserSchema.getSelfAnonUserInfo(
+     localDb,
+     anonProfileId,
+     selectedChannel?.id
+   );
+ 
+   setSelfAnonUserInfo(userInfo);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->